### ISSUE TITLE
segfault when json file absent

### ISSUE THREAD

**Issue by mcast** [wtsi-npg/baton#220]
This might be a distraction from #218, but

```
ubuntu@mca-icli427:~$ i-cgp -v /usr/local/bin/baton-metamod --operation add --file ~/batonq.json 
[d] IRODS_ENVIRONMENT_FILE=/home/ubuntu/.irods/irods_environment-cgp.json
[d] exec /usr/local/bin/baton-metamod
2020-03-31T11:11:37Z ERROR Failed to open '/home/ubuntu/batonq.json': error 2 No such file or directory
2020-03-31T11:11:37Z ERROR Processed 0 items with 0 errors
Segmentation fault (core dumped)
ubuntu@mca-icli427:~$ echo $?
139
```

This is with https://github.com/wtsi-npg/baton/releases/download/2.0.1/baton-2.0.1.tar.gz which I built with https://packages.irods.org/apt/pool/xenial/main/i/irods-dev/irods-dev_4.2.7_amd64.deb

`~/batonq.json` is absent because I deleted it. I ran the command because I wondered whether I could provoke #218 without bothering to put it back.

**Comment by mcast** [comment]
Sorry, might be a dup of #216

**Comment by kjsanger** [comment]
Fixed by  #216
